src/main/webapp/auth.js

```diff
@@ -55,16 +55,16 @@ function handleAuthenticationState() {
     signInButton.setAttribute('hidden', '');
     featureContainer.removeAttribute('hidden');
 
-    // Populate information panels at top of dashboard
-    populateGmail();
-    populateTasks();
-    populateCalendar();
+    // Populate information panels at top of dashboard asynchronously
+    new Promise(() => populateGmail());
+    new Promise(() => populateTasks());
+    new Promise(() => populateCalendar());
```

Comment:
Nit: I'm not super familiar with asynchronous programming in JavaScript, but it looks a bit weird to just create those promises like that...

Why have you decided not to make the `populate.*()` methods async?

Reply:
`populate.*()` methods are already "async" in that the calls to `fetch` return a promise and don't block.